src/applications/lgy/coe/containers/EligibilityApp.jsx

```diff
@@ -60,24 +59,30 @@ const EligibilityApp = props => {
         content = <CoePending />;
         break;
       default:
-        content = <LoadingIndicator message="Loading..." />;
+        content = <CoeNotApplicable />;
     }
   } else {
-    router.push('/introduction');
+    content = <CoeNotApplicable />;
   }
 
   return (
     <>
-      <header className="row vads-u-padding-x--1">
-        <FormTitle title="Apply for a VA home loan Certificate of Eligibility" />
-        <p>Request for a Certificate of Eligibility (VA Form 26-1880)</p>
-      </header>
-      {content}
+      <RequiredLoginView
+        serviceRequired={backendServices.USER_PROFILE}
+        user={props.user}
```

Comment:
It's mostly a knit, but would you mind destructuring the user prop up above to maintain consistency with the other props?